<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TotalTestSuite</class>
 <widget class="QWidget" name="TotalTestSuite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_Dir"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_SelectDir">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icon.qrc">
           <normaloff>:/icon/folder_open.ico</normaloff>:/icon/folder_open.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>29</width>
           <height>29</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Search">
         <property name="text">
          <string>查找</string>
         </property>
         <property name="icon">
          <iconset resource="icon.qrc">
           <normaloff>:/icon/icons8-view-64.png</normaloff>:/icon/icons8-view-64.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>29</width>
           <height>29</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Create">
         <property name="text">
          <string>创建</string>
         </property>
         <property name="icon">
          <iconset resource="icon.qrc">
           <normaloff>:/icon/icons8-add-file-100.png</normaloff>:/icon/icons8-add-file-100.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>29</width>
           <height>29</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidget_SearchReslut"/>
     </item>
     <item>
      <widget class="QLabel" name="label_InfoTip">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
